CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+<a name="7.2.4"></a>
+## [7.2.4](https://github.com/NativeScript/nativescript-angular/compare/7.2.3...7.2.4) (2019-04-23)
+
+
+### Bug Fixes
+
+* **router:** routing services should be provided in forRoot only ([#1729](https://github.com/NativeScript/nativescript-angular/issues/1729)) ([0f6a975](https://github.com/NativeScript/nativescript-angular/commit/0f6a975))
+* on destroy remove the lastBootstrappedModule ([5e13263](https://github.com/NativeScript/nativescript-angular/commit/5e13263))
```

Comment:
this and the next 3 commits are all part of one PR - https://github.com/NativeScript/nativescript-angular/pull/1728 could we link only this PR and/or the issue it is related to - https://github.com/NativeScript/nativescript-angular/issues/923 instead of listing all commits